ghost/admin/app/components/posts/debug.js

```diff
@@ -0,0 +1,162 @@
+                createdAt: batch.created_at ? moment(batch.created_at).format('DD MMM, YYYY, HH:mm:ss') : '',
+                segment: batch.member_segment || '',
+                providerId: batch.provider_id || null,
+                errorMessage: batch.error_message || '',
+                errorStatusCode: batch.error_status_code || '',
+                recipientCount: 0
+            };
+        });
+    }
+
+    get temporaryFailureData() {
+        return this.recipientFailures?.filter((failure) => {
+            return failure.severity === 'temporary';
+        }).map((failure) => {
+            return {
+                id: failure.id,
+                code: failure.code,
+                enhancedCode: failure.enhanced_code,
+                message: failure.message,
+                member: {
+                    name: failure.member?.name || '',
+                    email: failure.member?.email || '',
+                    initials: this.getInitials(failure.member?.name)
+                }
+            };
+        });
+    }
+
+    get permanentFailureData() {
+        return this.recipientFailures?.filter((failure) => {
+            return failure.severity === 'permanent';
+        }).map((failure) => {
+            return {
+                id: failure.id,
+                code: failure.code,
+                enhancedCode: failure.enhanced_code,
+                message: failure.message,
+                member: {
+                    name: failure.member?.name || '',
+                    email: failure.member?.email || '',
+                    initials: this.getInitials(failure.member?.name)
+                }
+            };
+        });
+    }
+
+
+    getInitials(name) {
+        if (!name) {
+            return 'U';
+        }
+        let names = name.split(' ');
+        let intials = names.length > 1 ? [names[0][0], names[names.length - 1][0]] : [names[0][0]];
+        return intials.join('').toUpperCase();
+    }
+
+
+    getStatusLabel(status) {
+        if (status === 'submitted') {
+            return 'Submitted';
+        } else if (status === 'submitting') {
+            return 'Submitting';
+        } else if (status === 'pending') {
+            return 'Pending';
+        } else if (status === 'failed') {
+            return 'Failed';
+        }
+        return status;
+    }
+
+    @action
+    loadData() {
+        this.fetchEmailBatches();
+        this.fetchRecipientFailures();
+    }
+
+    async fetchEmailBatches() {
+        try {
+            if (this._fetchEmailBatches.isRunning) {
+                return this._fetchEmailBatches.last;
+            }
+            return this._fetchEmailBatches.perform();
+        } catch (e) {
+            if (!didCancel(e)) {
+                // re-throw the non-cancelation error
+                throw e;
+            }
+        }
+    }
+
+    @task
+    *_fetchEmailBatches() {
+        let statsUrl = this.ghostPaths.url.api(`emails/${this.post.email.id}/batches`);
```

Comment:
?include=count.recipients is possible to get the total recipients counts

Reply:
✅